ui/app/routes/vault/cluster/secrets/backend/create-root.js

```diff
@@ -8,22 +8,10 @@ import { inject as service } from '@ember/service';
 import EditBase from './secret-edit';
 
 const secretModel = (store, backend, key) => {
-  const backendModel = store.peekRecord('secret-engine', backend);
-  const modelType = backendModel.modelTypeForKV;
-  // TODO kv engine cleanup - oish, this will need some thought.
-  if (modelType !== 'secret-v2') {
-    const model = store.createRecord(modelType, {
-      path: key,
-    });
-    return model;
-  }
-  const secret = store.createRecord(modelType);
-  secret.set('engine', backendModel);
```

Comment:
we're no longer setting the `engine` param for the secret model, is that right? could that be an issue?